aptos-move/framework/aptos-framework/sources/object.move

```diff
@@ -540,6 +546,10 @@ module aptos_framework::object {
         to: address,
     ) acquires ObjectCore {
         let owner_address = signer::address_of(owner);
+        assert!(
+            permissioned_signer::check_permission_exists(owner, TransferPermission { object }),
```

Comment:
So this is just true for a legacy signer?

Reply:
Correct